packages/components/src/components/table/table-head-row/table-head-row-styles.ts

```diff
@@ -7,6 +7,10 @@ export const getComponentCss = (): string => {
       ':host': addImportantToEachRule({
         display: 'table-row',
         ...hostHiddenStyles,
+        '&::before': {
+          // since "before" element counts as table cell, and we have "before" in table rows, we have to add it also to table-head-row to make it to have the same amount of cells
```

Comment:
I don't get it.

Reply:
@denyo if we remove this line of code, then it looks like this:
![Bildschirm_foto 2023-03-08 um 09 14 08](https://user-images.githubusercontent.com/2981884/223659736-c80e20be-18e3-45a8-9e7f-d2e2ac4668aa.jpg)

"::before" of table row counts as table cell, so the contents of the table is moved one column to the right.